.eslintrc

```
@@ -7,8 +7,17 @@
     "plugin:jsx-a11y/recommended"
   ],
   "plugins": ["jsx-a11y"],
+  "ignorePatterns": ["*.test.js"],
   "rules": {
     "react/no-unescaped-entities": "off",
-    "import/no-anonymous-default-export": "off"
+    "import/no-anonymous-default-export": "off",
+    "no-unused-vars": [
+      "error",
+      {
+        "vars": "all",
+        "args": "after-used",
+        "ignoreRestSiblings": false
```

O padrão de `ignoreRestSibling` já é `false` (conforme a [documentação](https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings)).

```suggestion
        "args": "after-used"
```